src/lib/containers/entity/page/title_bar/useGetEntityTitleBarProperties.tsx

```diff
@@ -0,0 +1,192 @@
+    enabled: !!(bundle?.entityType && isContainerType(bundle.entityType)),
+  })
+
+  // If this is the latest entity version, show the "versionless" DOI if it exists.
+  const useFallbackVersionlessDOI =
+    bundle &&
+    !bundle.doiAssociation &&
+    isVersionableEntity(bundle.entity) &&
+    bundle.entity.isLatestVersion
+  const { data: versionlessDOIAssociation } = useGetDOIAssociation(
+    entityId,
+    undefined,
+    'ENTITY',
+    {
+      enabled: useFallbackVersionlessDOI,
+    },
+  )
+
+  const dataFileHandle = bundle && getDataFileHandle(bundle)
+
+  const size =
+    dataFileHandle?.contentSize &&
+    calculateFriendlyFileSize(dataFileHandle.contentSize)
+  const fileHandleStorageInfo =
+    dataFileHandle && getFileHandleStorageInfo(dataFileHandle)
+  const storageLocation =
+    fileHandleStorageInfo &&
+    'location' in fileHandleStorageInfo &&
+    fileHandleStorageInfo.location
+  const endpoint =
+    fileHandleStorageInfo &&
+    'endpoint' in fileHandleStorageInfo &&
+    fileHandleStorageInfo.endpoint
+  const bucket =
+    fileHandleStorageInfo &&
+    'bucket' in fileHandleStorageInfo &&
+    fileHandleStorageInfo.bucket
+  const fileKey =
+    fileHandleStorageInfo &&
+    'fileKey' in fileHandleStorageInfo &&
+    fileHandleStorageInfo.fileKey
+  const externalUrl =
+    fileHandleStorageInfo &&
+    'url' in fileHandleStorageInfo &&
+    fileHandleStorageInfo.url
+
+  const md5 = dataFileHandle?.contentMd5
+  const downloadAlias =
+    bundle?.entity.name != bundle?.fileName && bundle?.fileName
+
+  const uploadDestinationString =
+    uploadDestination && getUploadDestinationString(uploadDestination)
+
+  // If there is no version-specific DOI, fall back to the versionless DOI
+  const doiAssociation = useFallbackVersionlessDOI
+    ? versionlessDOIAssociation
+    : bundle?.doiAssociation
+  const doi = doiAssociation && `https://doi.org/${doiAssociation?.doiUri}`
+
+  const containerItems = entityChildrenResponse?.totalChildCount
+
+  const datasetItems =
+    bundle?.entity && isEntityRefCollectionView(bundle.entity)
+      ? ((bundle?.entity as EntityRefCollectionView).items ?? []).length
+      : null
+
+  return [
+    {
+      key: 'id',
+      title: 'SynID',
+      value: <CopyToClipboardString value={entityId} />,
+    },
+    {
+      key: 'access',
+      title: 'Access',
+      value: <HasAccessV2 entityId={entityId} />,
+    },
+    size && { key: 'fileSize', title: 'Size', value: size },
+    containerItems != null && {
+      key: 'containerItems',
+      title: 'Items',
+      value: containerItems.toLocaleString(),
+    },
+    datasetItems != null && {
+      key: 'entityRefCollectionItems',
+      title: 'Items',
+      value: datasetItems.toLocaleString(),
+    },
+    doi && {
+      key: 'doi',
+      title: 'DOI',
+      value: (
+        <Link href={doi} rel={'noopener noreferrer'} target={'_blank'}>
+          {doi}
+        </Link>
+      ),
+    },
+    md5 && {
```

Comment:
This ordering is out-of-sync with the video in the PR description. Since the screen recording was uploaded, this was changed so the DOI appears before the MD5. The DOI should now always appear as one of the first 4 properties, so it will always be visible.